rts/at/integration/src/main/java/org/jboss/narayana/rest/integration/RecoveryManager.java

```diff
@@ -44,6 +44,11 @@
 
     private final Map<String, ParticipantDeserializer> deserializers = new ConcurrentHashMap<String, ParticipantDeserializer>();
 
+    /**
+     * A map of participants persisted in the object store, it maps 'Participant Id' to 'Uid'
+     */
+    private final Map<String, Uid> committedParticipants = new ConcurrentHashMap<>();
```

Comment:
Would persistedParticipants be clearer (though I do accept the store API is calling it committed)? Or are these really committed from a transactional perspective?

Reply:
+1